addons/binding/org.openhab.binding.dsmr/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,35 @@
+Automatic-Module-Name: org.openhab.binding.dsmr
+Manifest-Version: 1.0
+Bundle-ActivationPolicy: lazy
+Bundle-ClassPath: .
+Bundle-ManifestVersion: 2
+Bundle-Name: DSMR Binding
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.binding.dsmr;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.4.0.qualifier
+Export-Package: org.openhab.binding.dsmr,
+ org.openhab.binding.dsmr.handler
+Import-Package: 
+ gnu.io,
+ javax.measure,
+ javax.measure.quantity,
+ org.apache.commons.lang,
+ org.eclipse.jdt.annotation;resolution:=optional,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.core.common,
+ org.eclipse.smarthome.core.i18n,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.library.unit,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.thing.binding.builder,
+ org.eclipse.smarthome.core.thing.type,
+ org.eclipse.smarthome.core.types,
+ org.openhab.binding.dsmr,
+ org.openhab.binding.dsmr.handler,
+ org.osgi.framework,
+ org.slf4j,
+ tec.uom.se.quantity
```

Comment:
Afaik we should normally not import this, @htreu? This should be hidden as an implementation detail IIRC

Reply:
But when my quick scan of the sources is right its not needed here, so just drop.

Reply:
This was indeed not needed. I'll remove it.